app/assets/javascripts/components/audio_player.js.jsx

```diff
@@ -388,7 +388,7 @@ class TimeDisplay extends React.Component {
 
 TimeDisplay.propTypes = {
   elapsedTime: React.PropTypes.number,
-  duration: React.PropTypes.string
+  duration: React.PropTypes.number
```

Comment:
oooh good to know, that explains why it looked like it was coming through as a number in other areas. 👍